internal/releaseinfo/releaseinfo.go

```diff
@@ -12,8 +15,20 @@ type ReleaseInfo struct {
 func FetchLatestRelease() (*ReleaseInfo, error) {
 	url := "https://glasskube.dev/release.json"
 
-	resp, _ := http.Get(url)
-
+	resp, err := http.Get(url)
+	if err != nil {
+		if err := httperror.CheckResponse(resp); err != nil {
+			switch {
+			case httperror.IsNetworkError(err):
+				return nil, fmt.Errorf("Network Timeout, check your network %v: %v", url, err)
+			case httperror.Is(err, http.StatusServiceUnavailable):
+				return nil, fmt.Errorf("Could not connect to the glasskube server. Please check your internet connection and try again. %v: %v", url, err)
```

Comment:
Since this is a relatively low-level function, I think there should be no calls to action (e.g. "check your network") in the error messages. They can be added higher up where this function is used.